pkg/util/util_test.go

```diff
@@ -332,3 +334,383 @@ func TestJoinQualifiedName(t *testing.T) {
 		}
 	}
 }
+
+const gatewayfirst = `Iface	Destination	Gateway 	Flags	RefCnt	Use	Metric	Mask		MTU	Window	IRTT                                                       
+enp0s26u1u2u2	00000000	0100FE0A	0003	0	0	1024	00000000	0	0	0                                                                   
```

Comment:
a minor point, but can you rename the interface names so they are easier to read and so that the tables all align correctly?  Will make this easier to review